syntax = "proto3";

import "svg.proto";

package transport_catalogue_serialize;

message Offset{
	double x = 1;
	double y = 2;
}

message RenderSettings{
	double width = 1;
	double height = 2;
	double padding = 3;
	double line_width = 4;
	double stop_radius = 5;
	int32 bus_label_font_size = 6;
	Offset bus_label_offset = 7;
	int32 stop_label_font_size = 8;
	Offset stop_label_offset = 9;
	Color underlayer_color = 10;
	double underlayer_width = 11;
	repeated Color color_palette = 12;
}